cli/create/builtin_templates/templates/cartridge/test/helper.lua

```diff
@@ -0,0 +1,68 @@
+-- This file is required automatically by luatest.
```

Comment:
The stuff that adds the test environment for the cartridge application, I suggest that it also be moved to a separate commit.